proxy/context.go

```diff
@@ -102,27 +102,51 @@ const (
 
 	// IngressCertsPath is the path location for ingress certificates
 	IngressCertsPath = "/etc/istio/ingress-certs/"
-)
 
-// DefaultMeshConfig configuration
-func DefaultMeshConfig() proxyconfig.ProxyMeshConfig {
-	return proxyconfig.ProxyMeshConfig{
-		DiscoveryAddress:   "istio-pilot:8080",
-		EgressProxyAddress: "istio-egress:80",
+	// AuthCertsPath is the path location for mTLS certificates
+	AuthCertsPath = "/etc/certs/"
 
-		ProxyListenPort:        15001,
-		ProxyAdminPort:         15000,
+	// CertChainFilename is mTLS chain file
+	CertChainFilename = "cert-chain.pem"
+
+	// KeyFilename is mTLS private key
+	KeyFilename = "key.pem"
+
+	// RootCertFilename is mTLS root cert
+	RootCertFilename = "root-cert.pem"
+)
+
+// DefaultProxyConfig for individual proxies
+func DefaultProxyConfig() proxyconfig.ProxyConfig {
+	return proxyconfig.ProxyConfig{
+		ConfigPath:             "/etc/istio/proxy",
```

Comment:
/var/lib/istio/...

Reply:
Let's not change values. I specifically wanted to maintain the same golden test outputs to avoid breaking anything.

Reply:
Lets not change please.